src/ngx_http_lua_util.c

```diff
@@ -4561,4 +4564,34 @@ ngx_http_lua_ffi_bypass_if_checks(ngx_http_request_t *r)
     r->disable_not_modified = 1;
 }
 
+
+#if (NGX_HTTP_V3)
+void
+ngx_http_lua_resume_quic_ssl_handshake(ngx_connection_t *c)
```

Comment:
Do we need to comment here to describe why quic need extra resume SSL_do_handshake() function call?